src/python/pants/pantsd/pants_daemon.py

```diff
@@ -73,18 +74,23 @@ class RuntimeFailure(Exception):
 
   class Factory(object):
     @classmethod
-    def create(cls, bootstrap_options=None):
+    def create(cls, options_bootstrapper=None, bootstrap_options=None):
       """
+      :param OptionsBootstrapper options_bootstrapper: The bootstrapper of options, if available.
       :param Options bootstrap_options: The bootstrap options, if available.
       """
-      bootstrap_options = bootstrap_options or cls._parse_bootstrap_options()
+      options_bootstrapper = options_bootstrapper or OptionsBootstrapper()
+      bootstrap_options = bootstrap_options or options_bootstrapper.get_bootstrap_options()
       bootstrap_options_values = bootstrap_options.for_global_scope()
 
+      options, build_config = OptionsInitializer(options_bootstrapper).setup(init_logging=False)
```

Comment:
I believe that this will re-force options parsing... we're already passing in the bootstrap_options.

Would it be possible to just define the new option as a bootstrap option instead...?

Reply:
(...yes, I think so. Let's do that.)

Reply:
Moved to a bootstrap option, reverted all of the change around option parsing, I'll send that as a separate PR because I think there's stuff to be cleaned up there, but that should be separate :)